services/prometheus/prometheus_test.go

```diff
@@ -35,14 +37,48 @@ import (
 
 const configPath = "../../testdata/prometheus/prometheus.yml"
 
+// RoundTripFunc.
```

Comment:
https://github.com/vektra/mockery/issues/19
 there is an issue with generating mocks for standard lib.

 Option is replace http.Client with mocked type and mockery it.

 But do really need it?

 After replacing prometheus we can back to standard tests with real application, instead of mocks.